scripts/perf/scheduler_ops_metrics.py

```diff
@@ -178,6 +181,17 @@ def set_dags_paused_state(is_paused):
 
 
 def main():
+    global MAX_RUNTIME_SECS
+    if 1 < len(sys.argv):
```

Comment:
This is an un-expected way of expressing it. Could you swap it so that `len()` is on the left-hand-side of the operator please?

(Sorry to such a small nity-picky change)

Reply:
Sure! Switched them.

> (Sorry to such a small nity-picky change)

No worries, that's important for code quality and consistency. Thanks for the comment @ashb :)